templates/index.html

```diff
@@ -304,9 +304,9 @@ <h3>Featured snaps</h3>
                 <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" alt="">
               {% endif %}
             </span>
```

Comment:
Can you update the span above too please for consistency?

Reply:
Yes, done.